Formula/stormpath-cli.rb

```diff
@@ -14,65 +14,87 @@ class StormpathCli < Formula
+    sha256 "5570069e1b1769af1fe481de6dd1d3a388492acddd2cdad7a3bde145615d5caf"
   end
 
-  resource "six" do
-    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
-    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  resource "PyJWT" do
+    url "https://files.pythonhosted.org/packages/55/88/88d9590195a7fcc947501806f79c0918d8d3cdc6f519225d4efaaf3965e8/PyJWT-1.4.0.tar.gz"
+    sha256 "e1b2386cfad541445b1d43e480b02ca37ec57259fd1a23e79415b57ba5d8a694"
+  end
+
+  resource "pyquery" do
+    url "https://files.pythonhosted.org/packages/e0/4c/29d89a63446a1693798c7431188b75371921de4ea1506c06159e9eca23d6/pyquery-1.2.13.tar.gz"
+    sha256 "fbc95cf422ac79fa00c5107a2f33dff7dd106d6de569493bd938881b75d42e49"
   end
 
   resource "python-dateutil" do
-    url "https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-2.4.2.tar.gz"
-    sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
+    url "https://files.pythonhosted.org/packages/3e/f5/aad82824b369332a676a90a8c0d1e608b17e740bbb6aeeebca726f17b902/python-dateutil-2.5.3.tar.gz"
+    sha256 "1408fdb07c6a1fa9997567ce3fcee6a337b39a503d80699e0f213de4aa4b32ed"
   end
 
-  resource "pydispatcher" do
-    url "https://pypi.python.org/packages/source/P/PyDispatcher/PyDispatcher-2.0.5.tar.gz"
-    sha256 "5570069e1b1769af1fe481de6dd1d3a388492acddd2cdad7a3bde145615d5caf"
+  resource "requests" do
+    url "https://files.pythonhosted.org/packages/49/6f/183063f01aae1e025cf0130772b55848750a2f3a89bfa11b385b35d7329d/requests-2.10.0.tar.gz"
+    sha256 "63f1815788157130cee16a933b2ee184038e975f0017306d723ac326b5525b54"
   end
 
-  resource "isodate" do
-    url "https://pypi.python.org/packages/source/i/isodate/isodate-0.5.4.tar.gz"
-    sha256 "42105c41d037246dc1987e36d96f3752ffd5c0c24834dd12e4fdbe1e79544e31"
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/b3/b2/238e2590826bfdd113244a40d9d3eb26918bd798fc187e2360a8367068db/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  resource "stormpath" do
+    url "https://files.pythonhosted.org/packages/08/00/8566ef03476112f13120569816644f3af7e8cdc7a13cb776417c520eca8e/stormpath-2.4.0.tar.gz"
+    sha256 "d226aecf5535680b537a3f95fb383b19fe08312a031cb26fcb3aa3b68d1f1b66"
+  end
+
+  resource "termcolor" do
+    url "https://files.pythonhosted.org/packages/8a/48/a76be51647d0eb9f10e2a4511bf3ffb8cc1e6b14e9e4fab46173aa79f981/termcolor-1.1.0.tar.gz"
+    sha256 "1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b"
   end
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    resources.each do |r|
-      r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    %w[cssselect docopt isodate lxml oauthlib PyDispatcher PyJWT pyquery python-dateutil requests six stormpath termcolor].each do |r|
```

Comment:
Oh, I just use `poet` to grab the resource blocks and do the rest myself. Explains why I hadn't noticed this before.

I don't have a strong objection to being explicit but I've noticed generally with explicit blocks when people add/remove resources they initially forget to update the array. The line should be wrapped as close to ~80 characters as possible, as well, to avoid unnecessarily stretchy formulae.